### DISCUSSION TITLE
Installing a plugin on self-hosted docker

### DISCUSSION THREAD

**Discussion by Nebelfuchs** [pretix/pretix#3201] [Q&A] [answered]
Hi there,

I am struggling with installing a plugin on my docker setup for pretix. I did create my own image as stated in the docs but I am not sure what exactly to do here: `You can now use that image mypretix instead of pretix/standalone in your service file (see above). Be sure to re-build your custom image after you pulled pretix/standalone if you want to perform an update.`
Do I have to pull pretix/standalone again or how do I connect my image to the pretix container? Sorry for stupid questions... any help would be highly appreciated.

**Comment by vb-03** [comment, marked as answer]
When building your custom image, it will use the pretix/standalone as a "dependency" to build. 
So, when building, it will use the latest pretix/standalone version that is pulled to your machine, add the plugins, and create the image.

**Reply by Nebelfuchs** [reply]
Thanks a lot for clarifying! Do I need a new Dockerfile (and image) for every additional plugin or is there a way to install multiple plugins at once? How would the syntax in the Dockerfile be like? Can I just write them in a row? 
e.g.:
FROM pretix/standalone:stable
USER root
RUN pip3 install pretix-pages pretix-fontpack-free pretix-passbook
USER pretixuser
RUN cd /pretix/src && make production

**Reply by raphaelm** [reply]
I suggest you just try it. It should work :)
